src/schema/types.rs

```diff
@@ -497,6 +497,11 @@ impl ColumnPath {
   pub fn string(&self) -> String {
     self.parts.join(".")
   }
+
+  /// Returns slice of parts of the column path.
+  pub fn as_slice(&self) -> &[String] {
```

Comment:
nit: we can implement `AsRef<[String]>` for this purpose too.

Reply:
Do you mean adding imp for AsRef trait for ColumnPath and removing `as_slice`?